cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -182,6 +184,8 @@ spec:
                 replicated:
                   properties:
                     size:
+                      maximum: 10
```

Comment:
```suggestion
                      maximum: 4
```
? Is there a use case for a replica size of >4? 

See https://www.spinics.net/lists/ceph-users/msg54226.html

Reply:
Isn't this be  10?

Reply:
@sebastian-philipp

Reply:
I was just wondering, if we really should allow a replica size of >4. 

Nevertheless, this PR is of course (as far as I can tell) correct as it is.

Reply:
Can you merge it? @sebastian-philipp